src/figuresClasses.ts

```diff
@@ -1,19 +1,79 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Triangle {
+function isPositiveNumber(figure: object): never | void {
+  const numbers = Object.values(figure);
 
+  numbers.forEach((n): void => {
+    if (n <= 0) {
```

Comment:
Don't cut variable name

```suggestion
  numbers.forEach((number): void => {
    if (number <= 0) {
```